libs/core/algorithms/include/hpx/parallel/algorithms/make_heap.hpp

```diff
@@ -293,6 +293,33 @@ namespace hpx::parallel {
             }
         }
```

Comment:
Is the original implementation of `sift_down_range()` still needed?

Reply:
I don't think so, the only place where sift_down_range() is called has been overloaded. We can delete the original implementation.